pkgs/applications/office/libreoffice/darwin/update-utils.nix

```diff
@@ -0,0 +1,54 @@
+{ lib }:
+let
+  # extractLatestVersionFromHtml :: String -> String
+  extractLatestVersionFromHtml = htmlString:
```

Comment:
Why are we not using something easy to use like pup?

Reply:
I wanted to write the complete update-script in nix because it integrates much better with changes to the derivation. In addition, I wanted to try out a test-driven approach with nixt and I quite like it.

 If I were to write it in bash, pup would obviously be a good choice.

Reply:
update scripts are usually not written in bash for the reasons I already mentioned.